### ISSUE TITLE
In Debug mode, fetch images from the project folder

### ISSUE THREAD

**Issue by hartbit** [schibsted/layout#52]
When working on layouts, adding/editing images in the asset catalog forces to re-compile the application. In the layout system fetched images from the asset catalog in the project folder, we could avoid that.

**Comment by nicklockwood** [comment]
I've considered this. Would have been easy in the pre-asset catalog days. Still possible now, but a bit more work to write the xcasset json parser.

**Comment by hartbit** [comment]
Yes, I thought about it, and its definitely non-trivial. And you would have to keep it up-to-date to support whatever new Asset Catalog feature they add that allows specifying the conditions that govern when an image is used.
